packages/desktop-gui/src/specs/specs-store.js

```diff
@@ -8,15 +8,24 @@ import Folder from './folder-model'
 
 const pathSeparatorRe = /[\\\/]/g
 
-export const allSpecsSpec = new Spec({
+export const allIntegrationSpecsSpec = new Spec({
   name: 'All Specs',
```

Comment:
Should probably also say 'All Integration Specs' here to match the logic of component spec label, otherwise it's not totally accurate when you have both.